lib/export-path.js

```diff
@@ -13,12 +13,25 @@ module.exports = function exportPath(namespace, name, value) {
     var target = namespace;
 
     var i;
+    var j;
     var part;
 
+    var prev;
+    var keys;
+
     for (i = 0; i < parts.length; i += 1) {
         part = parts[i];
         if (i === parts.length - 1 && value !== undefined) {
-            target[part] = value;
+            if (target[part]) {
+                prev = target[part];
+                target[part] = value;
+                keys = Object.keys(prev);
```

Comment:
FYI this file is included in the loader output, so any features we use in here must be supported by the target browser. Using `Object.keys` means older browsers (like IE8) won't be able to consume Webpack bundles created using this loader unless they use a polyfill.

I think I'm OK with this though.

Reply:
Thank you.
OK, that's why the coding rule seemed so restrictive for me 🙄
Fair-enough point 🙇